### PR TITLE
guidelines: add missing captions (heads) to the guidelines chapter on Mensural Notation

### PR DESCRIPTION
**Pull request by annplaksin**
MEI Developer workshop 2024: ref #1217 

Changes are made for the mensural chapter. Do others feel responsible for the other chapters?

### CODE REVIEW

**Review by rettinghaus — DISMISSED** [review]
Thank you so much!

File: source/docs/05-mensural.xml [hunk 1]
```diff
@@ -165,33 +165,33 @@
                   </list>
                   <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the ‘voice’ staff renders the notes in the code snippet, and the ‘reference’ staff, together with the dotted bar lines, are shown to help to visualize the relative values of the notes in the ‘voice’ staff.</p>
                   <p>The following example illustrates an <hi rend="italic">alteration</hi> (the second <hi rend="italic">brevis</hi>) in <hi rend="italic">modus minor perfectus</hi>. Notice that the second <hi rend="italic">brevis</hi> has doubled its regular value, it has been altered, unlike the first one. <figure>
-                        <head/>
+                        <head>Example on alteration: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```
Comment (musicEnfanthen) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example on alteration (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

File: source/docs/05-mensural.xml [hunk 2]
```diff
@@ -165,33 +165,33 @@
...
                      </figure> The same applies to the examples that follow.</p>
                   <p>The following example illustrates an <hi rend="italic">imperfection</hi> (the two <hi rend="italic">longae</hi>) in <hi rend="italic">modus minor perfectus</hi> with the same <hi rend="italic">longa</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">longa</hi> sequence but with an additional <hi rend="italic">dot of division</hi> between the two <hi rend="italic">breves</hi> (see <ptr target="#dots"/> for more details). Notice that here the <hi rend="italic">longae</hi> have been imperfected, unlike the previous example in which they kept the perfect value indicated by the mensuration. <figure>
-                        <head/>
+                        <head>Example on imperfection: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```
Comment (musicEnfanthen) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example on imperfection (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

File: source/docs/05-mensural.xml [hunk 3]
```diff
@@ -165,33 +165,33 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/imperfection.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>The following example in <hi rend="italic">modus minor imperfectus</hi> illustrates the use of a dot of augmentation following the <hi rend="italic">longa</hi> (see <ptr target="#dots"/> for more details). Notice that the <hi rend="italic">longa</hi>, which is supposed to be imperfect according to the mensuration, has a perfect value due to the augmentation dot. <figure>
-                        <head/>
+                        <head>Example on augmentation: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```
Comment (musicEnfanthen) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example on augmentation (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

File: source/docs/05-mensural.xml [hunk 4]
```diff
@@ -165,33 +165,33 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/augmentation.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>Finally, the following example illustrates the <att>dur.quality</att> values related to the <hi rend="italic">Ars antiqua</hi> style, for <hi rend="italic">perfect modus</hi>, with the breve equivalents notated in the lower staff for reference (as in the previous examples). <figure>
-                        <head/>
+                        <head>Example on ars antiqua related values for perfect modus: The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.</head>
```
Comment (musicEnfanthen) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example on ars antiqua related values for perfect modus (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```

**Review by musicEnfanthen — COMMENTED** [review]
Thank you @annplaksin. Suggestion to move the secondary hints into parentheses.

**Comment by pe-ro** [comment]
Prepositions are difficult things. :-) "Example on" should be "Example of". Alternatively, use "Augmentation example" instead of "Example on augmentation".  Or, simplest of all, just use "Augmentation". The reader can figure out that it's an example based on the visual context.

File: source/docs/05-mensural.xml (hunk 1 above)
Comment (pe-ro) [reply source/docs/05-mensural.xml]: I suggest:
"Alteration: The bottom staff, together with the dotted barlines, is used to help visualize the durational values of the notes in the upper staff."

File: source/docs/05-mensural.xml (hunk 2 above)
Comment (pe-ro) [reply source/docs/05-mensural.xml]: I suggest:

"Example omitting <att>dur.quality</att> for default values provided by the mensuration." 
and 
"Imperfection: the bottom staff, together with the dotted barlines, is used to help visualize the durational values of the notes in the upper staff."

File: source/docs/05-mensural.xml (hunk 4 above)
Comment (pe-ro) [reply source/docs/05-mensural.xml]: I suggest:

"Ars antiqua-related values for perfect modus."

Comment (pe-ro) [reply source/docs/05-mensural.xml]: And so on ...

**Comment by annplaksin** [comment]
I made a few corrections. Thanks for the feedback!

File: source/docs/05-mensural.xml [hunk 5]
```diff
@@ -122,7 +122,7 @@
                </list>
                <p>
                   <figure>
-                     <head/>
+                     <head>The upper staff shows the different mensural note shapes and the lower staff shows the different mensural rests.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]: Captions in other chapters don't use end punctuations:

```suggestion
                     <head>The upper staff shows the different mensural note shapes and the lower staff shows the different mensural rests</head>
```

File: source/docs/05-mensural.xml [hunk 6]
```diff
@@ -142,7 +142,7 @@
                </p>
                <p>
                   <figure>
-                     <head/>
+                     <head>Encoding of Inflammatis invidia / Sicut de ligno / Victimae paschali detail from F-Pn 146, fol. 22r (https://gallica.bnf.fr/ark:/12148/btv1b8454675g/f55.image).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                     <head>Encoding of Inflammatis invidia / Sicut de ligno / Victimae paschali detail from F-Pn 146, fol. 22r (https://gallica.bnf.fr/ark:/12148/btv1b8454675g/f55.image)</head>
```

File: source/docs/05-mensural.xml [hunk 7]
```diff
@@ -165,33 +165,33 @@
                   </list>
                   <p>The last three values are to be used exclusively in <hi rend="italic">Ars antiqua</hi> mensural notation, where <val>maior</val> and <val>minor</val> refer to types of <hi rend="italic">semibreves</hi>, and <val>duplex</val> refers to a type of <hi rend="italic">longa</hi>. Examples of each of these six values are presented below. In these examples, the ‘voice’ staff renders the notes in the code snippet, and the ‘reference’ staff, together with the dotted bar lines, are shown to help to visualize the relative values of the notes in the ‘voice’ staff.</p>
                   <p>The following example illustrates an <hi rend="italic">alteration</hi> (the second <hi rend="italic">brevis</hi>) in <hi rend="italic">modus minor perfectus</hi>. Notice that the second <hi rend="italic">brevis</hi> has doubled its regular value, it has been altered, unlike the first one. <figure>
-                        <head/>
+                        <head>Example of alteration (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of alteration (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff)</head>
```

File: source/docs/05-mensural.xml [hunk 8]
```diff
@@ -165,33 +165,33 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/alteration.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>It is possible to omit the <att>dur.quality</att> attribute in a note when the performed duration of the note is the default value provided by the mensuration. In this case, the <hi rend="italic">longas</hi> are perfect, just as the mensuration (<hi rend="italic">perfect modus minor</hi>) indicates. Therefore, the <att>dur.quality</att> attribute can be omitted for the two <hi rend="italic">longas</hi>. <figure>
-                        <head/>
+                        <head>Example omitting <att>dur.quality</att> for default values provided by the mensuration.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example omitting <att>dur.quality</att> for default values provided by the mensuration</head>
```

File: source/docs/05-mensural.xml [hunk 9]
```diff
@@ -165,33 +165,33 @@
...
                      </figure> The same applies to the examples that follow.</p>
                   <p>The following example illustrates an <hi rend="italic">imperfection</hi> (the two <hi rend="italic">longae</hi>) in <hi rend="italic">modus minor perfectus</hi> with the same <hi rend="italic">longa</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">brevis</hi>-<hi rend="italic">longa</hi> sequence but with an additional <hi rend="italic">dot of division</hi> between the two <hi rend="italic">breves</hi> (see <ptr target="#dots"/> for more details). Notice that here the <hi rend="italic">longae</hi> have been imperfected, unlike the previous example in which they kept the perfect value indicated by the mensuration. <figure>
-                        <head/>
+                        <head>Example of imperfection (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of imperfection (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff)</head>
```

File: source/docs/05-mensural.xml [hunk 10]
```diff
@@ -165,33 +165,33 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/imperfection.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>The following example in <hi rend="italic">modus minor imperfectus</hi> illustrates the use of a dot of augmentation following the <hi rend="italic">longa</hi> (see <ptr target="#dots"/> for more details). Notice that the <hi rend="italic">longa</hi>, which is supposed to be imperfect according to the mensuration, has a perfect value due to the augmentation dot. <figure>
-                        <head/>
+                        <head>Example of augmentation (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of augmentation (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff)</head>
```

File: source/docs/05-mensural.xml [hunk 11]
```diff
@@ -165,33 +165,33 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/augmentation.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>Finally, the following example illustrates the <att>dur.quality</att> values related to the <hi rend="italic">Ars antiqua</hi> style, for <hi rend="italic">perfect modus</hi>, with the breve equivalents notated in the lower staff for reference (as in the previous examples). <figure>
-                        <head/>
+                        <head>Example of ars antiqua related values for perfect modus (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff.)</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of ars antiqua related values for perfect modus (The bottom staff, together with the dotted barlines, is used here to help visualizing the durational values of the notes in the upper staff)</head>
```

File: source/docs/05-mensural.xml [hunk 12]
```diff
@@ -165,33 +165,33 @@
...
                   <p>An alternative encoding---removing the <att>dur.quality</att> attributes for notes which lengths are not modified from their default values (<abbr>i.e.</abbr>, the perfect long and the minor <hi rend="italic">semibreves</hi>)---would be: <figure>
-                        <head/>
+                        <head>Encoding of the ars antiqua related values for perfect modus (see example above).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Encoding of the ars antiqua related values for perfect modus (see example above)</head>
```

File: source/docs/05-mensural.xml [hunk 13]
```diff
@@ -202,28 +202,28 @@
...
                   <p>
                      <figure>
-                        <head/>
+                        <head>Example of "partial imperfection of an immediate part" (ad partem propinquam).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of "partial imperfection of an immediate part" (ad partem propinquam)</head>
```

File: source/docs/05-mensural.xml [hunk 14]
```diff
@@ -202,28 +202,28 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/partial-imp-01-propinquam.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>Partial imperfection can also happen from both sides of a note at once, as shown below:</p>
                   <p>
                      <figure>
-                        <head/>
+                        <head>Example of "partial imperfection" from both sides (ad partes).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of "partial imperfection" from both sides (ad partes)</head>
```

File: source/docs/05-mensural.xml [hunk 15]
```diff
@@ -202,28 +202,28 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/partial-imp-02-bilateral.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>An example of partial imperfection caused by a note three orders apart is given next. Here the <hi rend="italic">longa</hi> is partially imperfected by a <hi rend="italic">minima</hi> (instead of by a <hi rend="italic">semibrevis</hi>).</p>
                   <p>
                      <figure>
-                        <head/>
+                        <head>Example of "partial imperfection of a remote part" (ad partem remotam).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of "partial imperfection of a remote part" (ad partem remotam)</head>
```

File: source/docs/05-mensural.xml [hunk 16]
```diff
@@ -202,28 +202,28 @@
...
                         <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="verovio code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/verovio/partial-imp-03-remotam.mei" parse="text"/></egXML>
                      </figure>
                   </p>
                   <p>In the next example, the <hi rend="italic">longa</hi> is also imperfected by a <hi rend="italic">minima</hi>. However, the <att>num</att> and <att>numbase</att> ratio is different from the example above because the default value of the <hi rend="italic">longa</hi> here (18 <hi rend="italic">minimas</hi>) is different from that of the previous example (12 <hi rend="italic">minimas</hi>).</p>
                   <p>
                      <figure>
-                        <head/>
+                        <head>Example of "partial imperfection of a remote part" (ad partem remotam).</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of "partial imperfection of a remote part" (ad partem remotam)</head>
```

File: source/docs/05-mensural.xml [hunk 17]
```diff
@@ -265,7 +265,7 @@
                <div xml:id="changeinmensuration" type="div3">
                   <head>Change in mensuration</head>
                   <p>The following example illustrates a <hi rend="italic">change in mensuration</hi>. In this case, the element <gi scheme="MEI">mensur</gi> is used within the <gi scheme="MEI">layer</gi> element, preceding the stream of notes affected by the new mensuration defined by it. <figure>
-                        <head/>
+                        <head>Example of a change in mensuration.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of a change in mensuration</head>
```

File: source/docs/05-mensural.xml [hunk 18]
```diff
@@ -274,7 +274,7 @@
...
-                        <head/>
+                        <head>Example of omitted mensuration signs.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                        <head>Example of omitted mensuration signs</head>
```

File: source/docs/05-mensural.xml [hunk 19]
```diff
@@ -308,7 +308,7 @@
                      <graphic url="../images/modules/mensural/ex_ligatures01.png"/>
                   </figure>
                   <figure>
-                     <head/>
+                     <head>Encoding of recta and obliqua ligatures.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                     <head>Encoding of recta and obliqua ligatures</head>
```

File: source/docs/05-mensural.xml [hunk 20]
```diff
@@ -319,7 +319,7 @@
                      <graphic url="../images/modules/mensural/ex_ligatures02.png"/>
                   </figure>
                   <figure>
-                     <head/>
+                     <head>Encoding of that ligature with more than two notes with recta and obliqua.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                     <head>Encoding of that ligature with more than two notes with recta and obliqua</head>
```

File: source/docs/05-mensural.xml [hunk 21]
```diff
@@ -329,7 +329,7 @@
                <p>The data organization based on <gi scheme="MEI">measure</gi> elements that usually prevails in MEI is not appropriate for mensural notation because most music until 1600 does not have measures (bars) as we understand them today. Even though it is not defined by the mensural module, a more suitable alternate data organization without measures is available: <gi scheme="MEI">staff</gi> elements may occur directly within the <gi scheme="MEI">section</gi> element without being organized into measures first. The organization of events (notes, rests, etc.) within the <gi scheme="MEI">staff</gi> and <gi scheme="MEI">layer</gi> elements remains unchanged.</p>
                <p>
                   <figure>
-                     <head/>
+                     <head>Example of an encoding without <gi scheme="MEI">measure</gi>.</head>
```
Comment (rettinghaus) [inline source/docs/05-mensural.xml]:
```suggestion
                     <head>Example of an encoding without <gi scheme="MEI">measure</gi></head>
```

**Review by rettinghaus — CHANGES_REQUESTED** [review]
Thanks. Looks almost perfect!

**Comment by annplaksin** [comment]
So, I committed all suggested changes. Thanks for reviewing.

**Review by rettinghaus — APPROVED** [review]
Thank you so much!

**Review by musicEnfanthen — APPROVED** [review]

**Comment by musicEnfanthen** [comment]
Thanks for the fix! 🙏

**Pull request merged**